provider/lxd/provider_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	gc "gopkg.in/check.v1"
 
 	"github.com/juju/juju/cloud"
+	lxdServer "github.com/juju/juju/container/lxd"
```

Comment:
this choice of renaming the package seems a bit odd to me, but I don't immediately have an obviously better name (containerLXD ?)